ibc-core/ics02-client/context/src/context.rs

```diff
@@ -102,3 +103,62 @@ pub trait ClientExecutionContext:
         height: Height,
     ) -> Result<(), ContextError>;
 }
+
+/// An optional trait that extends the client validation context capabilities by
+/// providing additional methods for validating a client state, particularly
+/// benefiting ICS-07 Tendermint clients by granting access to essential
```

Comment:
I think we shouldn't use « particularly benefitting ICS-07 » anymore. Maybe « mainly benefitting ICS-07 Tendermint clients and likes »?

Reply:
[274e0b1](https://github.com/cosmos/ibc-rs/pull/1166/commits/274e0b1c2e67f8a7f68ff23f1b3ce04cc5b16961)